README.md

```diff
@@ -44,7 +44,7 @@ Based on the number of unique contributors and repository forks, GitHub Octovers
 
 Angular Material has revised/improved its build processes. To preserve stability with applications currently using Angular Material, we will have three (3) types of releases:
 
-*  `major` :  this type of release will be the Angular 2.x efforts maintained in a separate repository called [Material2](https://github.com/angular/material2). This type of release will not be used within Angular Material 1.x.
+*  `major` :  this type of release will be the Angular efforts maintained in a separate repository called [Angular Material](https://github.com/angular/material2). This type of release will not be used within AngularJS Material.
```

Comment:
"Called Angular Material" is a bit odd here since the repo itself is still called material2. How about "Major releases will be done as part of the latest iteration of Angular Material on a [separate repo]".